src/modules/catalog.js

```diff
@@ -2087,22 +2098,22 @@ class Catalog {
    * @function replaceCatalog
    * @param {object} parameters - Additional parameters for catalog details
    * @param {string} parameters.section - The section to update
-   * @param {string} [parameters.notification_email] - An email address to receive an email notification if the task fails
+   * @param {string} [parameters.notificationEmail] - An email address to receive an email notification if the task fails
```

Comment:
Can we add backwards compatibility logic for `notification_email` in all of the functions that used it as well? Thank you!